pensjon-brevbaker-api-model/build.gradle.kts

```diff
@@ -10,7 +10,7 @@ plugins {
 }
 
 group = "no.nav.pensjon.brev"
-version = "122"
+version = "123"
```

Comment:
Her er kanskje meldingen fra Petter relevant? https://nav-it.slack.com/archives/C014U6RKY7M/p1739520574292339